### PR TITLE
docs: update kubernetes cs for pod security standards

### PR DESCRIPTION
**Pull request by rileydakota**
# You're A Rockstar

Thank you for submitting a Pull Request (PR) to the Cheat Sheet Series.

> :triangular_flag_on_post: If your PR is related to grammar/typo mistakes, please double-check the file for other mistakes in order to fix all the issues in the current cheat sheet.

Please make sure that for your contribution:

- [x] In case of a new Cheat Sheet, you have used the [Cheat Sheet template](https://github.com/OWASP/CheatSheetSeries/blob/master/templates/New_CheatSheet.md).
- [x] All the markdown files do not raise any validation policy violation, see the [policy](https://github.com/OWASP/CheatSheetSeries/actions?query=workflow%3A%22Markdown+Link+Check%22).
- [x] All the markdown files follow these [format rules](https://github.com/OWASP/CheatSheetSeries/blob/master/CONTRIBUTING.md#markdown).
- [x] All your assets are stored in the **assets** folder.
- [x] All the images used are in the **PNG** format.
- [x] Any references to websites have been formatted as `[TEXT](URL)`
- [x] You verified/tested the effectiveness of your contribution (e.g., the defensive code proposed is really an effective remediation? Please verify it works!).
- [x] The CI build of your PR pass, see the build status [here](https://github.com/OWASP/CheatSheetSeries/actions).

If your PR is related to an issue, please finish your PR text with the following line:

This PR covers issue #1041 

Thank you again for your contribution :smiley:

### CODE REVIEW

**Comment by rileydakota** [comment]
@szh I'll try to look at the failing build tonight

**Comment by szh** [comment]
> @szh I'll try to look at the failing build tonight

Thanks! It's just lint errors:

```
cheatsheets/Kubernetes_Security_Cheat_Sheet.md:400 MD036/no-emphasis-as-heading/no-emphasis-as-header Emphasis used instead of a heading [Context: "Utilize Pod Security Standards..."]
cheatsheets/Kubernetes_Security_Cheat_Sheet.md:410:283 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
```

File: cheatsheets/Kubernetes_Security_Cheat_Sheet.md [hunk 1]
```diff
@@ -413,6 +391,62 @@ spec:
...
+- **Restricted**: The most restrictive policy, designed to enforce current Pod hardening practices at the expense of some compatibility. Intended for security critical workloads or untrusted users. Restricted includes all of the enforcements from the baseline policy, in addition to much more restrictive requirements, such as requiring the dropping of all capabilities, enforcing runAsNotRoot, and more.
+
+Each of the profiles have defined settings baselines that can be found in more detail [here](https://kubernetes.io/docs/concepts/security/pod-security-standards/#profile-details).
+
+The Pod Security Admission Controller allows you to enforce, audit, or warn upon the violation of a defined policy. `audit` and `warn` modes can be utilized to determine if a particular Pod Security Standard would normally prevent the deployment of a pod when set to `enforce` mode.
```
Comment (mackowski) [inline cheatsheets/Kubernetes_Security_Cheat_Sheet.md:410]:
```suggestion
The Pod Security Admission Controller allows you to enforce, audit, or warn upon the violation of a defined policy. `audit` and `warn` modes can be utilized to determine if a particular Pod Security Standard would normally prevent the deployment of a pod when set to `enforce` mode.

```

File: cheatsheets/Kubernetes_Security_Cheat_Sheet.md [hunk 2]
```diff
@@ -413,6 +391,62 @@ spec:
 
 For more information on security context for Pods, refer to the documentation at <https://kubernetes.io/docs/tasks/configure-pod-container/security-context>
 
+--
+
+### Continuously assess the privileges used by containers
+
+We strongly recommend that all your containers should adhere to the principle of least privilege, since your security risk is heavily influenced by the capabilities, role bindings, and privileges given to containers. Each container should only have the minimum privileges and capabilities that allows it to perform its intended function.
+
+**Utilize Pod Security Standards and the Built-in Pod Security Admission Controller to enforce container privilege levels**
```
Comment (mackowski) [inline cheatsheets/Kubernetes_Security_Cheat_Sheet.md:400]:
```suggestion
#### Utilize Pod Security Standards and the Built-in Pod Security Admission Controller to enforce container privilege levels

```

File: cheatsheets/Kubernetes_Security_Cheat_Sheet.md [hunk 3]
```diff
@@ -413,6 +391,62 @@ spec:
...
+    pod-security.kubernetes.io/enforce: restricted
+    pod-security.kubernetes.io/audit: restricted
+    pod-security.kubernetes.io/warn: restricted
+```
+
+Cluster administrators should properly organize and and enforce policy on cluster namespaces, only permitting the privileged policy on namespaces where it is absolutely required, such as for critical cluster services that require access to the underlying host. Namespaces should be set to the lowest Pod Security Policy that can be enforced and supports their risk level.
+
+If more granular policy enforcement is required beyond the three profiles (Privileged, Baseline, Restricted), Third party admission controllers like OPA Gatekeeper or Kyverno, or built in Validating Admission Policy can be utilized.
+
+**Use Pod security policies to control the security-related attributes of pods, which includes container privilege levels.**
```
Comment (mackowski) [inline cheatsheets/Kubernetes_Security_Cheat_Sheet.md:429]:
```suggestion
#### Use Pod security policies to control the security-related attributes of pods, which includes container privilege levels.

```

File: cheatsheets/Kubernetes_Security_Cheat_Sheet.md [hunk 4]
```diff
@@ -413,6 +391,65 @@ spec:
...
+    pod-security.kubernetes.io/audit: restricted
+    pod-security.kubernetes.io/warn: restricted
+```
+
+Cluster administrators should properly organize and and enforce policy on cluster namespaces, only permitting the privileged policy on namespaces where it is absolutely required, such as for critical cluster services that require access to the underlying host. Namespaces should be set to the lowest Pod Security Policy that can be enforced and supports their risk level.
+
+If more granular policy enforcement is required beyond the three profiles (Privileged, Baseline, Restricted), Third party admission controllers like OPA Gatekeeper or Kyverno, or built in Validating Admission Policy can be utilized.
+
+#### Use Pod security policies to control the security-related attributes of pods, which includes container privilege levels.
+
```
Comment (szh) [inline cheatsheets/Kubernetes_Security_Cheat_Sheet.md:432]:
```suggestion
#### Use Pod security policies to control the security-related attributes of pods, which includes container privilege levels
```

File: cheatsheets/Kubernetes_Security_Cheat_Sheet.md [hunk 5]
```diff
@@ -413,6 +391,65 @@ spec:
...
+- **Restricted**: The most restrictive policy, designed to enforce current Pod hardening practices at the expense of some compatibility. Intended for security critical workloads or untrusted users. Restricted includes all of the enforcements from the baseline policy, in addition to much more restrictive requirements, such as requiring the dropping of all capabilities, enforcing runAsNotRoot, and more.
+
+Each of the profiles have defined settings baselines that can be found in more detail [here](https://kubernetes.io/docs/concepts/security/pod-security-standards/#profile-details).
+
+The Pod Security Admission Controller allows you to enforce, audit, or warn upon the violation of a defined policy. `audit` and `warn` modes can be utilized to determine if a particular Pod Security Standard would normally prevent the deployment of a pod when set to `enforce` mode.
+
+
```
Comment (szh) [inline cheatsheets/Kubernetes_Security_Cheat_Sheet.md:413]:
```suggestion
```

File: cheatsheets/Kubernetes_Security_Cheat_Sheet.md [hunk 6]
```diff
@@ -413,6 +391,65 @@ spec:
...
 
+--
+
+### Continuously assess the privileges used by containers
+
+We strongly recommend that all your containers should adhere to the principle of least privilege, since your security risk is heavily influenced by the capabilities, role bindings, and privileges given to containers. Each container should only have the minimum privileges and capabilities that allows it to perform its intended function.
+
+#### Utilize Pod Security Standards and the Built-in Pod Security Admission Controller to enforce container privilege levels
+
+
```
Comment (szh) [inline cheatsheets/Kubernetes_Security_Cheat_Sheet.md:402]:
```suggestion
```

File: cheatsheets/Kubernetes_Security_Cheat_Sheet.md [hunk 7]
```diff
@@ -413,6 +391,62 @@ spec:
...
+  name: policy-test
+  labels:    
+    pod-security.kubernetes.io/enforce: restricted
+    pod-security.kubernetes.io/audit: restricted
+    pod-security.kubernetes.io/warn: restricted
+```
+
+Cluster administrators should properly organize and and enforce policy on cluster namespaces, only permitting the privileged policy on namespaces where it is absolutely required, such as for critical cluster services that require access to the underlying host. Namespaces should be set to the lowest Pod Security Policy that can be enforced and supports their risk level.
+
+If more granular policy enforcement is required beyond the three profiles (Privileged, Baseline, Restricted), Third party admission controllers like OPA Gatekeeper or Kyverno, or built in Validating Admission Policy can be utilized.
```
Comment (szh) [inline cheatsheets/Kubernetes_Security_Cheat_Sheet.md:427]:
```suggestion
If more granular policy enforcement is required beyond the three profiles (Privileged, Baseline, Restricted), Third party admission controllers like OPA Gatekeeper or Kyverno, or built-in Validating Admission Policy can be utilized.

```

**Comment by szh** [comment]
@rileydakota I'd like to get this in so I'm going ahead and fixing the lint issues.

File: cheatsheets/Kubernetes_Security_Cheat_Sheet.md [hunk 8]
```diff
@@ -413,6 +391,63 @@ spec:
...
+    pod-security.kubernetes.io/enforce: restricted
+    pod-security.kubernetes.io/audit: restricted
+    pod-security.kubernetes.io/warn: restricted
+```
+
+Cluster administrators should properly organize and and enforce policy on cluster namespaces, only permitting the privileged policy on namespaces where it is absolutely required, such as for critical cluster services that require access to the underlying host. Namespaces should be set to the lowest Pod Security Policy that can be enforced and supports their risk level.
+
+If more granular policy enforcement is required beyond the three profiles (Privileged, Baseline, Restricted), Third party admission controllers like OPA Gatekeeper or Kyverno, or built-in Validating Admission Policy can be utilized.
+
+
```
Comment (szh) [inline cheatsheets/Kubernetes_Security_Cheat_Sheet.md:429]:
```suggestion
```

**Review by szh — APPROVED** [review]

**Comment by szh** [comment]
@mackowski we should definitely squash when we merge this b/c of all the GH commits

**Review by mackowski — APPROVED** [review]

**Pull request merged**